K_deployment.md

```diff
@@ -8,6 +8,15 @@ Now that we've got our application prepared for production, getting it running i
 MIX_ENV=prod PORT=4001 iex -S mix phoenix.server
 ```
 
+To run the application in detached mode
+
+```elixir
+MIX_ENV=prod PORT=4001 elixir --detached -S mix phoenix.server
+```
+
+Running the application in detached mode allows you to keep the application running even after you terminate your shell connection with the server.
```

Comment:
One convention we have maintained in the guides is to substitute "we" and "our" for "you" and "your". It gives the subtle feeling that we're all in this together.

Making those substitutions here would be great, and I apologize for not having a contribution guide up yet. That would have covered this.